src/entity.cpp

```diff
@@ -5766,13 +5766,9 @@ bool Entity::isMobile()
 		return false;
 	}
 
-	if ( behavior == &actMonster && 
-		(introstage == 9
-		|| introstage == 11 + MOVIE_MIDGAME_BAPHOMET_HUMAN_AUTOMATON
-		|| introstage == 11 + MOVIE_MIDGAME_BAPHOMET_MONSTERS
-		|| introstage == 11 + MOVIE_MIDGAME_HERX_MONSTERS) )
+	if ( behavior == &actMonster && MainMenu::isCutsceneActive() )
 	{
-		return false; // mid-game crawls.
+		return false;
```

Comment:
Verify this is working correctly? I.e don't get slaughtered by demons in hell while watching cutscene in network lobby

Reply:
It works fine but I added another if check to actMonster(). it returns immediately `if (movie || MainMenu::isCutsceneActive())`